build.gradle

```diff
@@ -98,7 +98,7 @@ dependencies {
     compile 'org.apache.pdfbox:pdfbox:2.0.14'
     compile 'org.apache.pdfbox:fontbox:2.0.14'
     compile 'org.apache.pdfbox:xmpbox:2.0.14'
-
+    compile group: 'org.mariadb.jdbc', name: 'mariadb-java-client', version: '2.4.0'
```

Comment:
Please group this with the other db-related dependencies (mysql/postgre).
Can the mysql dependency be deleted now?

Reply:
Will do.